pyproject.toml

```diff
@@ -82,3 +85,81 @@ ignore_missing_imports = true
 
 [tool.scriv]
 categories = ["Features", "Bugfixes", "Documentation"]
+
+[tool.tox]
```

Comment:
Given this is a pretty large chunk maybe better to have it in a `tox.ini`? That way you'd actually get syntax highlighting/checking on this too which might be nicer.

Reply:
Good point, I'll do that!